src/CareTogether.Api/OData/LiveODataModelController.cs

```diff
@@ -362,17 +369,91 @@ private async Task<LiveModel> RenderLiveModelAsync()
             var organizationId = userDataAccessScope.OrganizationId;
             var anonymize = userDataAccessScope.Anonymize;
 
-            var result = await cache.GetOrAddAsync(
-                $"LiveODataModelController-RenderLiveModelAsync-{organizationId}-{(anonymize ? "ANON" : "PII")}",
-                async cacheEntry =>
-                {
-                    var result = await RenderLiveModelInternalAsync(organizationId, anonymize);
+            var isGlobal = userDataAccessScope.IsGlobal;
 
-                    cacheEntry.SlidingExpiration = TimeSpan.FromMinutes(1);
-                    cacheEntry.AbsoluteExpiration = DateTimeOffset.Now.AddMinutes(5);
+            var organizationIds = isGlobal
+                ? await accountsResource.GetValidOrganizationsAsync()
+                : [organizationId];
+
+            var results = await organizationIds
+                .Select(async organizationId =>
+                    await cache.GetOrAddAsync(
+                        $"LiveODataModelController-RenderLiveModelAsync-{organizationId}-{Guid.NewGuid()}-{(anonymize ? "ANON" : "PII")}",
```

Comment:
Why include the `{Guid.NewGuid()}` here? Won't that make it so the cache is *never* hit?